full-service/src/service/wallet.rs

```diff
@@ -733,7 +733,7 @@ mod tests {
         let account_id = account_obj.get("account_id").unwrap().as_str().unwrap();
         assert_eq!(
             account_id,
-            "b266572c325f5f0388e4645cfa945d8527e90a11bf2182c28f62090225e73138"
+            "f9957a9d050ef8dff9d8ef6f66daa608081e631b2d918988311613343827b779"
```

Comment:
Looking into why this changed.

Reply:
This changed because the public address fields changed (https://github.com/mobilecoinfoundation/mobilecoin/pull/711), with the field change from `fog_authority_fingerprint_sig` to `fog_authority_sig`.

Note, the account_id is created from the main subaddress's public address: https://github.com/mobilecoinofficial/full-service/blob/main/full-service/src/db/account.rs#L38